app/src/pages/market_home_page.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import { MarketHomeContainer } from '../components'
+import { ConnectedWeb3, useConnectWeb3 } from '../hooks/connectedWeb3'
+
+const MarketHomePage = () => {
+  useConnectWeb3()
```

Comment:
The home page is the one that doesn't have to have this. Otherwise it doesn't make sense to have the "Connect" buttons.

Reply:
Mmmmm right here we will have a problem, how do we solve the filtering by owneship? Maybe we can connect the user when clicks the filter, what do you think ?

Reply:
We can render different components when the user is connected and when she's not.

Reply:
I create  two components, one connected, another disconnected, also I create another context for the case in which the user is disconnected